contract_sale_invoicing/tests/test_contract_sale_invoicing.py

```diff
@@ -45,3 +45,10 @@ def test_contract_sale_invoicing_without(self):
         self.sale_order.action_confirm()
         self.contract.recurring_create_invoice()
         self.assertEqual(self.sale_order.invoice_status, 'to invoice')
+
+    def test_multi_contract_recurring_create_invoice(self):
+        self.contract.copy()
+        self.contract.copy()
+        self.contract.copy()
+        self.contract.copy()
+        self.env['contract.contract'].cron_recurring_create_invoice()
```

Comment:
make an assert here

Reply:
But Which values, I called only Scheduled Actions(cron_recurring_create_invoice) not any return here

Reply:
check that the cron create correctly the invoices for all contracts